server/core/auth.js

```diff
@@ -152,7 +152,7 @@ module.exports = {
    */
   checkAccess(user, permissions = [], page = false) {
     const userPermissions = user.permissions ? user.permissions : user.getGlobalPermissions()
-
+    const checkPermissions = permissions
```

Comment:
Use `permissions` directly instead of creating a new copy. `_.intersection` does not mutate the original, so it's perfectly safe to use it directly.

Reply:
For some reason I thought there was going to be a scoping issue. I just retried it though and I was mistaken.